accounting/entries.go

```diff
@@ -231,15 +231,65 @@ func sweepEntries(tx lndclient.Transaction, u entryUtils) ([]*HarmonyEntry, erro
 	return []*HarmonyEntry{txEntry, feeEntry}, nil
 }
 
+// isUtxoManagementTx checks whether a transaction is restructuring our utxos.
+func isUtxoManagementTx(txn lndclient.Transaction) bool {
+	// Check all inputs.
+	for _, input := range txn.PreviousOutpoints {
+		if !input.IsOurOutput {
+			return false
+		}
+	}
+
+	// Check all outputs.
+	for _, output := range txn.OutputDetails {
+		if !output.IsOurAddress {
+			return false
+		}
+	}
+
+	// If all inputs and outputs belong to our wallet, it's utxo management.
+	return true
+}
+
+// createOnchainFeeEntry creates a fee entry for an on chain transaction.
+func createOnchainFeeEntry(
```

Comment:
Can you please take a look at the formatting guideline doc I shared? Or the surrounding code? This isn't how we want our function definitions to be formatted.
Alternatively I can also push a fix commit to your branch if you don't want to deal with it.

Reply:
I missed the link in your prior comment. I've pushed an update with that style fix. Shame that `make fmt` doesn't handle that properly.

Reply:
Thanks!

> Shame that `make fmt` doesn't handle that properly.

Yeah, definitely. One of the biggest downsides of Go is that there is no configurable formatter such as `prettier`...